index.html

```diff
@@ -2434,6 +2434,59 @@ <h3>
           </p>
         </aside>
       </section>
+      <section class="informative">
+        <h3>
+          Web Apps with scope conflicts
+        </h3>
+        <p>
+          It is possible for two web apps to be installed where there are
+          conflicts between the scope. These conflicts come with disadvantages
+          outlined below and they are generally not recommended.
+        </p>
+        <ul>
+          <li>The scopes of the two web apps can be on the same origin. Not
```

Comment:
Do you think the "Not recommended" (and similar) bits should be _emphasized_?

Reply:
We should avoid RFC2119 language in non-normative sections. 

Instead, we should be clear on what the implications are. 

I think this list can go as it's covered by the paragraph suggested above.